src/client/src/Domains/Jobs/api/api.ts

```diff
@@ -11,49 +11,6 @@ export type jobType =
 
 export type targetYearsType = 'Freshman' | 'Sophmore' | 'Junior' | 'Senior';
 
-export const departments = [
```

Comment:
This is removed because this data was moved to a different file in a previous PR, but it was forgotten to be removed form here.

Reply:
Acknowledged.